CHANGES.md

```diff
@@ -1,5 +1,15 @@
 # Change log for kotlinx.coroutines
 
+## Version 1.6.2
+
+* Fixed a bug with `ThreadLocalElement` not being correctly updated when the most outer `suspend` function was called directly without `kotlinx.coroutines` (#2930).
+* Fixed multiple data races that might have been affecting `Mutex` and `runBlocking` event loop (#3250, #3251).
+* Obsolete `TestCoroutineContext` is removed, which fixes the `kotlinx-coroutines-test` JPMS package being split between `kotlinx-coroutines-core` and `kotlinx-coroutines-test` (#3218).
+* Updated the ProGuard rules to further shrink the size of the resulting DEX file with coroutines (#3263). Thanks @agrieve!
```

Comment:
```suggestion
* Updated the ProGuard rules to further shrink the size of the resulting DEX file with coroutines (#3111). Thanks, @agrieve!
```
We usually reference the issues, right?

Reply:
I was thinking the PR is more insightful. I've left both references